test/security/test_authenticator.py

```diff
@@ -15,10 +12,11 @@
                                          InvalidTokenError,
                                          NonDecodableTokenError,
                                          InvalidRedirectUriError)
+from azul_test_case import AlwaysTearDownTestCase
 from retorts import AuthResponseHelper, TestKeyManager
 
 
-class AuthenticatorTestCase(TestCase):
+class AuthenticatorTestCase(AlwaysTearDownTestCase):
```

Comment:
And why this one?

Reply:
I assume you overlooked this. I'm just gonna remove it.

Reply:
Woops.. Yes I did.